### DISCUSSION TITLE
A "custom window size/place" feature

### DISCUSSION THREAD

**Discussion by Stooovie** [rxhanson/Rectangle#1363] [General]
I like the "Almost maximize" feature, but I'd like even smaller window size with a keyboard shortcut. Sort of square-ish window in the middle of the screen. It would be great if user could specify their own window size/placement and add keyboard shortcut to it. the [Loop app](https://github.com/MrKai77/Loop) does have that feature. Thanks!

**Comment by rxhanson** [comment]
Thanks for the feedback. You can adjust the "Almost Maximize" size using the terminal:
https://github.com/rxhanson/Rectangle/blob/main/TerminalCommands.md#almost-maximize

Or you can add an extra centering command with a custom size, right below it in that same file:
https://github.com/rxhanson/Rectangle/blob/main/TerminalCommands.md#add-an-extra-centering-command-with-custom-size

**Comment by Stooovie** [comment]
Percentages would be better. Would that be possible? Because static dimensions mean vastly different results on a builtin screen and external display. I don't want that.
